### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #36: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_example_app.py. (closes #36)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: examples/app.py [hunk 1]
```diff
@@ -66,8 +74,6 @@
     SQLALCHEMY_TRACK_MODIFICATIONS=True,
     INDEXER_DEFAULT_INDEX=index_name,
     INDEXER_DEFAULT_DOC_TYPE='testrecord-v1.0.0',
-    SQLALCHEMY_DATABASE_URI=os.getenv('SQLALCHEMY_DATABASE_URI',
```
Comment (jirikuncar) [inline examples/app.py]: please keep it so people can use different databases with their example app.

File: examples/app-teardown.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+DIR=`dirname "$0"`
+
+cd $DIR
+export FLASK_APP=app.py
+
```
Comment (jirikuncar) [inline examples/app-teardown.sh]: we should include also `db destroy --yes-i-know`

File: examples/app.py (hunk 1 above)
Comment (hachreak) [inline examples/app.py]: done

File: examples/app-teardown.sh (hunk 2 above)
Comment (hachreak) [inline examples/app-teardown.sh]: done

**Comment by hachreak** [comment]
:shipit: @jirikuncar ?

**Pull request merged**